### PR TITLE
LICENSE & README are required by artifacthub.io, so un-.helmignoring

### PR DESCRIPTION
**Pull request by jkremser**
`LICENSE & README` are used on artifacthub.io so putting them back

Signed-off-by: Jirka Kremser <jiri.kremser@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -410,7 +410,7 @@ test-failover:
 terratest: # Run terratest suite
 	@$(eval RUNNING_CLUSTERS := $(shell k3d cluster list --no-headers | grep $(CLUSTER_NAME) -c))
 	@if [ $(RUNNING_CLUSTERS) != 3 ] ; then \
-		echo "$(RED)Make sure the you run the tests againt 3 running clusters$(NC)" ;\
+		echo "$(RED)Make sure you run the tests against 3 running clusters$(NC)" ;\
```
Comment (somaritane) [inline Makefile:413]: Let's make it a separate PR.
I know that the change is small, but it's unrelated to the scope of actual PR.

**Review by somaritane — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (jkremser) [reply Makefile:413]: k

**Review by somaritane — APPROVED** [review]
